.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "c_src/libs/c_reusable_data_structures"]
+	path = c_src/libs/c_reusable_data_structures
+	url = https://github.com/emqx/c_reusable_data_structures.git
+	branch = master
```

Comment:
still, better to pin a tag instead

Reply:
But it's just a tracking branch, the submodule itself is pinned by commit hash in the git object itself.
https://github.com/emqx/jq/pull/49/files/9d87fffa3e07085ca4b71fe7a5df8cf32cde1450#diff-13821eeef0636f9fc5f8a0677131978922278043a982629aba9869556721fe18